### ISSUE TITLE
Apply genart filter for Collections on koda.art

### ISSUE THREAD

**Issue by vikiival** [kodadot/nft-gallery#10991]
### Is your feature request related to a problem?

Currently we see the all collections on Polkadot assethub in koda.art platform.
This is not nice because we would like to see only genart

### Describe the solution you would like

if you visit `(subdomain).koda.art` there should be a default filter applied for collection explore `{ kind_eq: genart }`

```graphql
{
  collectionEntities(where: { kind_eq: genart }) {
    id
    name
    kind
    meta {
      kind
    }
  }
}
```

### Describe alternatives you have considered

I have considered to add this also to NFTentities 
lmk @kodadot/internal-dev 

Also this could be added as optional to existing queries so lmk if you need help with that

### Screenshots

![Screenshot 2024-09-17 at 09 22 25](https://github.com/user-attachments/assets/b3481511-8805-4bab-a441-df0c22ce216a)

### Are there opened related issues?

- https://github.com/kodadot/ops-internal/issues/1912

**Comment by vikiival** [comment]
This is available only on `ahp` yet,
- `base`, `mnt` will be released today
- `ahk` will be released this week

**Comment by Jarsen136** [comment]
> I have considered to add this also to NFTentities lmk 

Yes, please add it.
And maybe also some other queries like `collectionEntitiesConnection` `nftEntitiesConnection`

**Comment by vikiival** [comment]
> And maybe also some other queries like `collectionEntitiesConnection`

make sure u are using [v14](https://kodadot.squids.live/speck/v/v14/graphql)

it is there 

```grapgql
{
  collectionEntitiesConnection(where: { kind_eq: genart }, orderBy: id_ASC) {
    totalCount
  }
}

```

**Comment by vikiival** [comment]
Which are the cases you would need that for `nftEntitiesConnection` ? 
I am thiking you add it to `nftEntities` too but would like to hear if it's a valid case

**Comment by Jarsen136** [comment]
> Which are the cases you would need that for `nftEntitiesConnection` ? 

`queries/profileStatsById.graphql`  
```
profileStatsById {  listed: nftEntitiesConnection, sold: nftEntitiesConnection ... }
```
Display the correct stats of account, which only shows the stats related to genart.

> I am thiking you add it to `nftEntities` too but would like to hear if it's a valid case

On items explore page, filter nfts that is related to genart maybe.

**Comment by vikiival** [comment]
you can apply it from the collection

```grapgql
{
  obtained: nftEntitiesConnection(where: {currentOwner_eq: $id, burned_eq: false, metadata_not_eq: "", issuer_not_in: $denyList, collection: {kind_eq: genart}}, orderBy: name_ASC) {
    totalCount
  }
}

```

**Comment by Jarsen136** [comment]
> you can apply it from the collection
> 
> ```graphql
> {
>   obtained: nftEntitiesConnection(where: {currentOwner_eq: $id, burned_eq: false, metadata_not_eq: "", issuer_not_in: $denyList, collection: {kind_eq: genart}}, orderBy: name_ASC) {
>     totalCount
>   }
> }
> ```

Oh, I see. That's a good idea.

**Comment by Jarsen136** [comment]
On the items explore page in ahp network, it's using `tokenEntityList` resolver. 
Is it possible to add `kind` type there? @vikiival

**Comment by hassnian** [comment]
linking #10763

**Comment by Jarsen136** [comment]
> This is available only on `ahp` yet,
> 
> * `base`, `mnt` will be released today
> * `ahk` will be released this week

Let me know if you have graphql links for testing other networks.

like ahp:  https://ahp.gql.api.kodadot.xyz/ -> https://kodadot.squids.live/speck/v/v14/graphql

**Comment by vikiival** [comment]
mnt should be updated, base had some issues. 
ahk is failing because of cf

**Comment by vikiival** [comment]
> On the items explore page in ahp network, it's using `tokenEntityList` resolver.

I've tried - https://github.com/kodadot/stick/pull/308

**Comment by vikiival** [comment]
@Jarsen136 
- flock v3 
- basick v13
- speck v14

Still trying to fix stick

**Comment by vikiival** [comment]
Also @Jarsen136 
- https://github.com/kodadot/stick/pull/308
was merged and should be available in speck v14 lmk if it produces correct results

**Comment by Jarsen136** [comment]
> Also @Jarsen136
> 
> * [feat/token entity kind stick#308](https://github.com/kodadot/stick/pull/308)
>   was merged and should be available in speck v14 lmk if it produces correct results

It produces the intern error : )
<img width="989" alt="image" src="https://github.com/user-attachments/assets/70a5322a-0ef9-4f6b-bf85-9427c36dc230">

**Comment by Jarsen136** [comment]
cc @vikiival

**Comment by vikiival** [comment]
> cc @vikiival 

Can you please try to fix the SQL query by opening PR? 

You do not need to index it :)

**Comment by vikiival** [comment]
@Jarsen136 need any help/input from my side?

**Comment by Jarsen136** [comment]
> @Jarsen136 need any help/input from my side?

Yes. What's the graphql endpoint url for the `ahk` network?

**Comment by vikiival** [comment]
> What's the graphql endpoint url for the ahk network?

https://kodadot.squids.live/stick/v/v12/graphql 

> all except stick are already running prod version

**Comment by Jarsen136** [comment]
> > What's the graphql endpoint url for the ahk network?
> 
> https://kodadot.squids.live/stick/v/v12/graphql
> 

Look like there are still some issues on ahk @vikiival 

<img width="948" alt="image" src="https://github.com/user-attachments/assets/0baef968-a657-45a4-8e6a-403027464271">

**Comment by vikiival** [comment]
Yes this is related to the fact that no drops contain genart kind now

**Comment by Jarsen136** [comment]
> Yes this is related to the fact that no drops contain genart kind now

Then I would disable art gen mode for ahk network for now, as it defaults on.
